samples/.eslintrc.yml

```diff
@@ -1,3 +1,4 @@
 ---
 rules:
   no-console: off
+  no-unused-vars: off
```

Comment:
@JustinBeckwith this allows us to create variables in the samples that don't require using. In this case, it helps the users see how to get a reference to the `topics` or `subscriptions` we return. We don't need to use the variables, though.

So, I allowed it here (	samples/`), and disallowed it in `samples/system-test`.